stdlib/universe/holt_winters/holt_winters.go

```diff
@@ -245,65 +228,65 @@ func (r *HoltWinters) forecast(params []float64, onlyFit bool) []float64 {
 		phiH += math.Pow(phi, float64(t))
 
 		if r.seasonal {
-			seasonals[(t+so)%m] = sT
+			params.Set(seasonalsStart+(t+so)%m, sT)
 			so++
 		}
 
 		if onlyFit || (r.includeFitData && t < l) || t >= l {
-			fcast = append(fcast, yT)
+			fcast.Append(yT)
 		}
 	}
 	return fcast
 }
 
 // Compute sum squared error for the given parameters.
-func (r *HoltWinters) sse(params []float64) float64 {
+func (r *HoltWinters) sse(params *mutable.Float64Array) float64 {
 	sse := 0.0
 	fcast := r.forecast(params, true)
-	defer r.tracker.free(len(fcast))
-	for i := 0; i < len(fcast); i++ {
+	defer fcast.Release()
```

Comment:
This is fine, that array is a new one every time that `forecast` gets called and should be released every time. It would go out of scope anyways at the end of `sse`.